### PR TITLE
Default value for expand can be 'help'

### PR DESCRIPTION
**Pull request by ihorskyi**
#456

### CODE REVIEW

**Comment by SBoudrias** [comment]
This wouldn't display the prompt expanded, it would just default to expand if the user press enter. Is that correct? Is that what you want?

**Comment by ihorskyi** [comment]
@SBoudrias yep, that's it

**Comment by SBoudrias** [comment]
How about we just make help the default option if no `default` key is provided?

**Comment by ihorskyi** [comment]
@SBoudrias That would make even more sense

**Comment by SBoudrias** [comment]
@ihorskyi cool, I think changing that will make this review good to merge.

**Comment by ihorskyi** [comment]
@SBoudrias great, will change that tomorrow.

File: lib/prompts/expand.js [hunk 1]
```diff
@@ -223,10 +223,14 @@ Prompt.prototype.validateChoices = function (choices) {
  */
 Prompt.prototype.generateChoicesString = function (choices, defaultIndex) {
   var defIndex = 0;
```
Comment (SBoudrias) [inline lib/prompts/expand.js]: Just change this to `this.opt.choices.length - 1;`

File: lib/prompts/expand.js [hunk 2]
```diff
@@ -222,11 +222,15 @@ Prompt.prototype.validateChoices = function (choices) {
...
  */
 Prompt.prototype.generateChoicesString = function (choices, defaultIndex) {
-  var defIndex = 0;
+  var defIndex = this.opt.choices.length - 1;
+  var defStr = this.opt.choices.pluck('key');
+
   if (_.isNumber(defaultIndex) && this.opt.choices.getChoice(defaultIndex)) {
     defIndex = defaultIndex;
+  } else if (_.isUndefined(defaultIndex)) {
+    defIndex = defStr.length - 1;
```
Comment (SBoudrias) [inline lib/prompts/expand.js]: You don't need this anymore

File: lib/prompts/expand.js [hunk 3]
```diff
@@ -222,11 +222,13 @@ Prompt.prototype.validateChoices = function (choices) {
  * @return {String} The rendered choices key string
  */
 Prompt.prototype.generateChoicesString = function (choices, defaultIndex) {
-  var defIndex = 0;
+  var defStr = this.opt.choices.pluck('key');
```
Comment (SBoudrias) [inline lib/prompts/expand.js]: I see you reverted the use of `this.opt.choices.length`, any reason for that? Doesn't look like the pluck is useful here as we only want the length.

File: lib/prompts/expand.js (hunk 2 above)
Comment (ihorskyi) [inline lib/prompts/expand.js]: @SBoudrias this was changed in following commit, it is outdated now

File: lib/prompts/expand.js (hunk 3 above)
Comment (ihorskyi) [inline lib/prompts/expand.js]: @SBoudrias The reason to use `this.opt.choices.pluck('key')` is to get choices with `keys` excluding separators. 
`this.opt.choices.length` === 6 
`defStr.length` === 5

Comment (SBoudrias) [inline lib/prompts/expand.js]: There's `choices.realLength`

**Comment by SBoudrias** [comment]
Great, thanks!

**Pull request merged**